website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -123,231 +164,3 @@ Please don't forget to add a new static-client on your OIDC provider settings wi
 - `-c`, `--private-key-password`:   Private key password. If the private key is encrypted using password
 - `-u`, `--username`:               Dashboard admin username
 - `-v`, `--version`:                Weave GitOps Enterprise version to install
-
-
-#### External DNS
```

Comment:
shall we provide the user with full example, or it's his responsibility to set it up ?

Reply:
Refactored the domain section. added link to https://github.com/kubernetes-sigs/external-dns but i guess that we could leave it to them to take this part. I could think that adding this annotation should be configurable